inspirehep/modules/workflows/templates/workflows/tickets/conflicts-ticket-template.html

```diff
@@ -0,0 +1,37 @@
+{#
+## This file is part of INSPIRE.
+## Copyright (C) 2020 CERN.
+##
+## INSPIRE is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## INSPIRE is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+Merge conflict needs to be resolved.
+
+https://{{ server_name }}/editor/holdingpen/{{ workflow_id }}
+
+{% if recid %}
+Record ID (recid): {{ recid|safe }}
+{% endif %}
+
+{% if arxiv_ids %}
+arXiv IDS: {{ arxiv_ids|safe }}
+{% endif %}
+
+{% if dois %}
+DOIs: {{ dois|safe }}
+{% endif %}
+
+{% if report_numbers %}
+Report numbers: {{ report_numbers|safe }}
+{% endif %}
```

Comment:
I don't think we need all of this, it's enough to put the data in the subject.

Reply:
Ah, ok then :+1: